docs/src/main/paradox/grpc.md

```diff
@@ -40,6 +40,14 @@ Akka Projections require Akka $akka.version$ or later, see @ref:[Akka version](o
 
 @@project-info{ projectId="akka-projection-grpc" }
 
+It is currently only possible to use @extref:[akka-persistence-r2dbc](akka-persistence-r2dbc:projection.html) as the
+projection storage and journal for this module.
+
+@@dependency [sbt,Maven,Gradle] {
+group=com.lightbend.akka
+artifact=akka-persistence-r2dbc_$scala.binary.version$
```

Comment:
missing `akka-projection-r2dbc`?

Reply:
Added in https://github.com/akka/akka-projection/pull/757/commits/8f4e6371dd44c95206712d2a5098f3ab9535071a